### PR TITLE
Change dekorate template to use the intended knative object

### PR DESCRIPTION
**Pull request by claudio4j**
The dekorate issue is fixed and use the correct knative spec template as per the removed comment.
https://github.com/dekorateio/dekorate/issues/869

### CODE REVIEW

**Review by gsmet — DISMISSED** [review]
Thanks for the PR!

I will let @iocanel review the Dekorate change but I have a comment about the config.

File: extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KnativeConfig.java [hunk 1]
```diff
@@ -463,15 +463,15 @@ public EnvVarsConfig getEnv() {
      * This value controls the minimum number of replicas each revision should have.
      * Knative will attempt to never have less than this number of replicas at any point in time.
      */
-    @ConfigItem
+    @ConfigItem(name = "min-scale")
```
[the remark this answers is not in the document]
Comment (claudio4j) [reply extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KnativeConfig.java]: The CI check had failed expecting `min-scale`, then I changed to kebab case locally and it worked, but I reverted it now, cleaned the build, it seems to be working, let's wait the CI.
Thanks for the heads up.

**Comment by gsmet** [comment]
@iocanel this one looks like a good candidate for the LTS too.

**Review by iocanel — APPROVED** [review]
LGTM

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**